SteamBundleSitesExtension.js

```diff
@@ -1093,19 +1165,12 @@ const siteHandlers = {
             bundleSitesBoxHandler.reveal(handler, $('.sr-unredeemed-steam-button'));
         });
         $('.SBSE_BtnRetrieve').click(() => {
-            const keys = [];
-
-            $('.sr-redeemed-bubble .keyfield-text').each((index, element) => {
-                const $game = $(element);
-                const $heading = $game.closest('[class^=sr-key]').prev().children().eq(0);
-
-                keys.push({
-                    key: $game.text().trim(),
-                    title: $heading.text().trim(),
-                });
-            });
-
-            bundleSitesBoxHandler.retrieve(keys);
+            bundleSitesBoxHandler.retrieve(extractKeys());
+        });
+        $('.SBSE_BtnExport').click(() => {
+            let product_json =JSON.parse(/window.models.product_json = ({.*});/.exec(document.head.innerHTML)[1]);
+            let bundleTitle = product_json.human_name;
```

Comment:
`<meta name="title" content="Humble Down Under Bundle">`
I think it's easier to get bundle name from this meta tag

Reply:
Wow, you're right.
It will fix in next commit.